### PR TITLE
Move call to proxy outside (webapi 2.8)

### LINKED ISSUE #1364: impossible to create a copy of PLP
1) Try to copy any plp(prediction)

expected:The copy has been created

actual:
front:
image

 back:
WebAPI/prediction/{id}/copyRequest Method: GETStatus Code: 500{"payload":{"cause":null,"stackTrace":[],"message":"An exception ocurred: org.springframework.core.convert.ConversionFailedException","localizedMessage":"An exception ocurred: org.springframework.core.convert.ConversionFailedException","suppressed":[]},"headers":{"id":"249a112a-4afe-32c3-1543-b30a203dac25","timestamp":1573718841739}}

as a result - the copy has been created, but it's unable for editing

### PR DESCRIPTION
**Pull request by ssuvorov-fls**
fixes #1364

### CODE REVIEW

File: src/main/java/org/ohdsi/webapi/prediction/PredictionController.java [hunk 1]
```diff
@@ -100,25 +110,31 @@ public void delete(@PathParam("id") int id) {
   @Produces(MediaType.APPLICATION_JSON)
   @Consumes(MediaType.APPLICATION_JSON)
   public PredictionAnalysisDTO createAnalysis(PredictionAnalysis pred) {
-
-    return conversionService.convert(service.createAnalysis(pred), PredictionAnalysisDTO.class);
+    PredictionAnalysis analysis = service.createAnalysis(pred);
+    // Before conversion entity must be refreshed to apply entity graphs
```
Comment (pavgra) [inline src/main/java/org/ohdsi/webapi/prediction/PredictionController.java]: The 3 lines of code are repeated multiple times. Why don't you extract them to a separate method, something like
```
reloadAndConvert() {
  // comment
  getById
  conversionService.convert
}
```

Same with pathways and estimations class

**Comment by anthonysena** [comment]
@olga-ganina please let me know if your team plans to address the feedback left by @pavgra on this PR. I agree with @pavgra's feedback - having this function centralized would be better than a copy/paste across many classes.

**Comment by olga-ganina** [comment]
@anthonysena  requested changes were implemented so we will proceed with review of this one

**Comment by anton-abushkevich** [comment]
already in master via https://github.com/OHDSI/WebAPI/pull/1479

**Pull request closed without merging**